src/wasm/types.rs

```diff
@@ -279,3 +286,10 @@ pub struct EthSignature {
     pub s: Vec<u8>,
     pub v: Vec<u8>,
 }
+
+#[derive(serde::Serialize)]
+pub struct BtcSignMessageSignature {
+    pub sig: Vec<u8>,
+    pub recid: u8,
+    pub electrum_sig65: Vec<u8>,
+}
```

Comment:
no need to replicate it here, right? the struct already exists in btc::SignMessageSignature.

Reply:
You're right, I replicated it since ETH had a similar struct there, but it may not be necessary

Reply:
When I remove this, I get into errors and compiler can not find `btc::SignMessageSignature`, so I kept it

Reply:
Fixed it, the error was due to the serialization definition for the struct